docs/framework/data/adonet/ado-net-technology-options-and-guidelines.md

```diff
@@ -22,7 +22,7 @@ The ADO.NET Data Platform is a multi-release strategy to decrease the amount of
  Entity SQL is a text-based query language designed to interact with an Entity Data Model. Entity SQL is an SQL dialect that contains constructs for querying in terms of higher-level modeling concepts, such as inheritance, complex types, and explicit relationships. Developers can also use Entity SQL directly with Object Services. For more information, see [Entity SQL Language](../../../../docs/framework/data/adonet/ef/language-reference/entity-sql-language.md).  
   
 ### EntityClient  
- EntityClient is a new .NET Framework data provider used for interacting with an Entity Data Model. EntityClient follows the .NET Framework data provider pattern of exposing <xref:System.Data.EntityClient.EntityConnection> and <xref:System.Data.EntityClient.EntityCommand> objects that return an <xref:System.Data.EntityClient.EntityDataReader>. EntityClient works with the Entity SQL language, providing flexible mapping to storage-specific data providers. For more information, see [EntityClient and Entity SQL](https://msdn.microsoft.com/library/49202ab9-ac98-4b4b-a05c-140e422bf527).  
+ EntityClient is a new .NET Framework data provider used for interacting with an Entity Data Model. EntityClient follows the .NET Framework data provider pattern of exposing <xref:System.Data.EntityClient.EntityConnection> and <xref:System.Data.EntityClient.EntityCommand> objects that return an <xref:System.Data.EntityClient.EntityDataReader>. EntityClient works with the Entity SQL language, providing flexible mapping to storage-specific data providers.  
```

Comment:
Maybe replace link 49202ab9-ac98-4b4b-a05c-140e422bf527 with https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/ef/entityclient-provider-for-the-entity-framework?